Client/Frontend/Strings.swift

```diff
@@ -1668,6 +1668,45 @@ extension String {
                     tableName: "Settings",
                     value: "Default Search Engine",
                     comment: "Accessibility label for default search engine setting.")
+                public static let LearnAboutSuggestions = MZLocalizedString(
+                    key: "Settings.Search.Accessibility.LearnAboutSuggestions.v121",
```

Comment:
string freeze for 121 has alread gone out, so this should be labelled as 122, as well
```suggestion
                    key: "Settings.Search.Accessibility.LearnAboutSuggestions.v122",
```

Reply:
oops, i meant 122